web/src/components/organisms/Project/Content/ContentDetails/hooks.ts

```diff
@@ -11,14 +12,23 @@ import { useT } from "@reearth-cms/i18n";
 
 import useContentHooks from "../hooks";
 
-type Props = {
-  itemId: string | undefined;
-};
-
-export default ({ itemId }: Props) => {
+export default () => {
   const { currentModel, itemsData } = useContentHooks();
+  const navigate = useNavigate();
+  const { projectId, workspaceId, itemId, modelId } = useParams();
   const t = useT();
 
+  const handleNavigateToModel = useCallback(
+    (modelId: string) => {
+      navigate(`/workspace/${workspaceId}/project/${projectId}/content/${modelId}`);
+    },
+    [navigate, workspaceId, projectId],
+  );
+
+  const handleNavigateBack = useCallback(() => {
+    navigate(`/workspace/${workspaceId}/project/${projectId}/content/${modelId}`);
+  }, [navigate, workspaceId, projectId, modelId]);
+
```

Comment:
These are the same so can you just use the handleNavigateToModal for the onBack as well?

Reply:
@KaWaite , merged the two into one.